common/Toolbox.ts

```diff
@@ -1,3 +1,4 @@
+import { BooleanTypeAnnotation } from "@babel/types";
```

Comment:
Is this used?

Reply:
Nope, probably an auto-import from me accidentally capitalizing `Boolean` somewhere!